asyncapi/github/docs/testing_guide.md

```
@@ -0,0 +1,149 @@
+
+### 5. Releases
+- **Published**: Go to **Releases** > **Create a new release**, enter a tag version and title, and click **Publish release**.
+
+### 6. Labels & Milestones
+- **Label Created**: Go to **Issues** > **Labels** > **New label**.
+- **Milestone Created**: Go to **Issues** > **Milestones** > **New milestone**.
+
+## Step 6: Verify the Results
+
+Check your Ballerina service logs. You should see the output from your remote functions confirming the receipt of the events:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)